engine/src/available-command.ts

```diff
@@ -377,17 +377,10 @@ export function possibleFreeActions(engine: Engine, player: Player) {
 
   engine.player(player).emit("freeActionChoice", pool);
 
-  // freeActions for Terrans / Itars during gaiaPhase
-  if (engine.phase === Phase.RoundGaia && (pl.canGaiaTerrans() || pl.canGaiaItars())) {
+  // freeActions for Terrans during gaiaPhase
+  if (engine.phase === Phase.RoundGaia && pl.canGaiaTerrans()) {
     if (pl.canGaiaTerrans()) {
       pool = freeActionsTerrans;
-    } else if (pl.canGaiaItars()) {
-      pool = freeActionsItars;
-      commands.push({
-        name: Command.Decline,
-        player,
-        data: { offers: [new Offer(Resource.TechTile, new Reward(4, Resource.GainTokenGaiaArea).toString())] },
-      });
     }
 
     burnDisabled = true;
```

Comment:
The terrans free action should probably be moved as well to `possibleGaiaFreeActions` - which should not call `possibleFreeActions` anymore. We can remove the `burnDisabled`  variable as well